x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -54,6 +89,11 @@ export class ForecastsTable extends Component {
   static contextType = context;
 
   componentDidMount() {
+    this.loadForecasts();
+    this.canDeleteJobForecast = checkPermission('canForecastJob');
```

Comment:
we should add a new `canDeleteForecast` capability

Reply:
Added in d29de18790177bc0c0d512a43845f70518ab6dc7